utils/workspace.js

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//@ts-check
+
+/**
+ * Use the following command to typescheck this file:
+ * npx tsc --target es2020  --watch --checkjs --noemit --moduleResolution node workspace.js
```

Comment:
if you use vscode it will type-check it by default for you without running a command!

Reply:
This is precisely for those of us who don't!